packages/sfui/frameworks/nuxt/playground/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "private": true,
+  "name": "@storefront-ui/nuxt-playground"
```

Comment:
What it this playground? What do we use it for? Only local development? Or maybe somewhere in the documentation?

Reply:
local development, it's used to test the module and also generate the types used for development. it's part of the nuxt module boilerplate.